src/Data/Nat/Coprimality.agda

```diff
@@ -87,63 +89,63 @@ coprime? i j with mkGCD i j
 -- Nothing except for 1 is coprime to 0.
 
 0-coprimeTo-m⇒m≡1 : ∀ {m} → Coprime 0 m → m ≡ 1
-0-coprimeTo-m⇒m≡1 {m} c = c (m ∣0 , ∣-refl)
+0-coprimeTo-m⇒m≡1 {m} coprime = coprime (m ∣0 , ∣-refl)
 
-¬0-coprimeTo-2+ : ∀ {n} → ¬ Coprime 0 (2 + n)
+¬0-coprimeTo-2+ : ∀ {n} → ¬ Coprime 0 (2+ n)
```

Comment:
This should use `.{{ NonTrivial n }}` instead of `2+ n`?

Reply:
So.. should also be renamed/deprecated?

Reply:
Interestingly, making this change caused `make test` to fail in `Data.Rational.Base`, with the solution involving some rather horrible explicit `instance` reasoning. One to debug... later?